README.rst

```diff
@@ -1,26 +1,69 @@
-****
-Home
-****
+openlifu
+========
 
-Open-pyFUS is a toolbox for planning and controlling focused ultrasound treatments. It generally replicates the functionality developed in the MATLAB `open-TFUS toolbox <https://github.com/OpenwaterHealth/opw_neuromod_sw>`_.
+|Actions Status| |Documentation Status|
+
+|PyPI version| |PyPI platforms|
+
+|GitHub Discussion|
```

Comment:
For this badge we'd have to enable discussions on the repo. Do we want that? It's pretty nice, but if there's some other central forum to which we want to direct discussions that we can replace this badge and point there instead.

Reply:
We could add another badge for discord also, next to the github discussions one